manageability-docker/build-az-container.sh

```diff
@@ -8,14 +8,14 @@ rm -rf "$DIR"/output
 mkdir -p "$DIR"/output
 cd ../inbm
 
-DOCKER_BUILDKIT=1 ./build.sh
```

Comment:
is buildkit breaking the build?

Reply:
@gblewis1 : Yeah it does. Having an issue with 'RUN gem install --no-ri --no-rdoc fpm -v 1.11.0' when running with DOCKER_BUILDKIT